pkg/util/kubernetes/pod.go

```diff
@@ -93,6 +93,10 @@ func natsPodReloaderContainer(image, tag, pullPolicy string) v1.Container {
 			constants.PidFilePath,
 		},
 	}
+	if authFilePath != "" {
+		container.Command = append(container.Command, "-authfile", authFilePath)
```

Comment:
I might have been wrong on this one... it looks like the reloader would already follow the events in the folder so if the volume is mounted underneath the main nats config file, it would signal as well in case of changes:

https://github.com/nats-io/nats-operator/blob/master/pkg/reloader/reloader.go#L94-L99

Reply:
Also, it's not supposed to be recursively watching.  (According to the documentation)

I created another PR to fix these problems with reloader independently.

Reply:
That's right... it only works per folder, it is not recursive so would need to list each of the folders that should be tracked for changes.